itou/asp/forms.py

```diff
@@ -18,15 +22,83 @@ def formfield_for_birth_place(*, with_birthdate_field, **kwargs):
         required=False,
         widget=RemoteAutocompleteSelect2Widget(
             attrs={
-                "data-ajax--url": reverse("autocomplete:communes"),
+                "data-ajax--url": SimpleLazyObject(lambda: reverse("autocomplete:communes")),
```

Comment:
```suggestion
                "data-ajax--url": reverse_lazy("autocomplete:communes"),
```
(avec `from django.urls import reverse_lazy` ;))